### ISSUE TITLE
API Gateway v1 protocol causes 500 response from API gateway (possible regression in 3.46.0)

### ISSUE THREAD

**Issue by jamestelfer** [DataDog/datadog-lambda-js#162]
## Expected Behaviour

Given an API Gateway integration with the v1 protocol in use, a simple Lambda function should return 200 OK on a valid route.

## Actual Behaviour

API gateway returned 500. With no change to any other configuration, changing the API Gateway to use protocol version 2 fixed the issue.

Using the function handler directly (without Datadog), the same function code responds with 200 OK for both protocol versions.

## Steps to Reproduce the Problem

  1. Create a Lambda with the following handler code:
      ```javascript
      module.exports.handler = async event => {
        return {
            "isBase64Encoded": false,
            "statusCode": 200,
            "headers": { "Content-Type": "application/json" },
            "multiValueHeaders": { },
            "body": "{}"
        }
      }
      ```
  1. Configure the Lambda with the v3.46.0 layer, and the appropriate handler and environment config.
  1. Connect the created Lambda to an HTTP API Gateway with the integration configured to use protocol version 1.

## Specifications

  - Datadog Lambda Layer version: 3.46.0
  - Node version: 12

## Workaround

Downgrade the layer to the previous release (v3.44.0)

**Comment by DylanLovesCoffee** [comment]
Hey @jamestelfer thanks for letting us know . We're looking into the issue and we'll keep you posted on what we find.

**Comment by DylanLovesCoffee** [comment]
@jamestelfer thanks again for raising this. It should be fixed now with release v3.48.0. I'll close the issue out but let us know if you run into anything!
